pyveda/vedaset/store/vedabase.py

```diff
@@ -8,7 +8,7 @@
 from pyveda.vedaset.base import BaseDataSet, BaseSampleArray
 from pyveda.vedaset.interface import SerializedVariableArray, PartitionedIndexArray, ArrayTransformPlugin
 from pyveda.frameworks.batch_generator import VedaStoreGenerator
-from pyveda.vv.labelizer import Labelizer
+from pyveda import vv
```

Comment:
why change the import line to import the whole vv module instead of Labelizer object?

Reply:
I can't remember why I did this... Will look and see if its actually necessary